doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-doc-var-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='create-doc-var']
+
+= Creating a document variable in the process
+After you set the document marshalling strategy, create a document variable in the related process. This variable is required for the document to be visible in the *Process Instances* view in {CENTRAL}.
+
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click on your project's name to open the asset view and click on the business process name.
+. Click on the canvas and click image:getting-started/diagram_properties.png[] on the right side of the window to open the *Diagram properties* panel.
+. Expand *Process Data* and click image:getting-started/6176.png[] and enter the following values:
++
+* Name: `document`
+* Custom Type: `org.jbpm.document.Document`
```

Comment:
I think it's worth also explaining how to create fields that can store a collection of documents using the 'org.jbpm.document.DocumentCollection' type. (https://github.com/kiegroup/kie-docs/pull/1569/files)